crates/wasmtime/src/lib.rs

```diff
@@ -427,6 +424,13 @@ pub use crate::trap::*;
 pub use crate::types::*;
 pub use crate::values::*;
 
+/// A convenience wrapper for `Result<T, anyhow::Error>`.
+///
+/// This type can be used to interact with `wasmtimes`'s extensive use
+/// of `anyhow::Error` while still not directly depending on `anyhow`.
+/// This type alias is identical to `anyhow::Result`.
+pub type Result<T> = std::result::Result<T, anyhow::Error>;
```

Comment:
Could this be `pub use anyhow::{Result, Error}` perhaps? One key feature this is currently missing is the `E` generic defaulting to `anyhow::Error` but still allowing two generics.